var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -422,9 +423,13 @@ def edit(self, spec, prefix):
         ldflags = []
         libs = []
 
-        # CP2K Makefile doesn't set C standard, but the source code uses
-        # C99-style for-loops with inline definition of iterating variable.
-        cflags.append(self.compiler.c99_flag)
+        # CP2K Makefile doesn't set C standard
+        if spec.satisfies("@2023.2:"):
+            # Use of DBL_DECIMAL_DIG
+            cflags.append(self.compiler.c11_flag)
```

Comment:
Is this still useful given the patch?

Reply:
This one should stay, it's for the other build system